### PR TITLE
Add "useGuest" to configuration at TeamCity Configuration.

### PR DESCRIPTION
**Pull request by Alea81**
Use "guestAuth" endpoint in TeamCity.js when "useGuest" configuration property is enabled.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,19 +160,21 @@ Supports the [TeamCity](https://www.jetbrains.com/teamcity/) build service.
...
+    "password": "teamcity_password",
+    "useGuest": true
   }
 }
 ```
 
 | Setting                 | Description
 |-------------------------|-----------------------------------------------------------------------------------------
-| `url`                   | The url to the TeamCity server (including the credentials without a trailing backslash).
+| `url`                   | The url to the TeamCity server (including the credentials without a trailing backslash, if not the guest user is used).
```
Comment (marcells) [inline README.md]: Is that comment really needed? Does it make a difference if `/guestAuth` or `/httpAuth` is included in the url?

Comment (marcells) [reply README.md]: Ah, forget about it. It doesn't have to do anything about the url formatting. ;-)

**Comment by marcells** [comment]
Thank you. :+1:

**Comment by Alea81** [comment]
You're welcome :)

**Pull request merged**